src/plugins/post-install/index.rst

```diff
@@ -0,0 +1,4 @@
+Post-Install Plugins
+====================
+
+.. automodule:: cylc.flow.post_install
```

Comment:
>  Is there a way to do that?

https://github.com/cylc/cylc-flow/pull/4142/files#r607893197

Reply:
Looks like Napoleon was not enabled on this repo, not sure why. One line change to enable it - https://github.com/metomi/rose/blob/380315e4785b7319ad3b73d985afbcd2fad858fe/sphinx/conf.py#L38

Reply:
If I add Napoleon, I start getting errors, possibly because we need https://github.com/agronholm/sphinx-autodoc-typehints

But I guess this isn't needed for this PR, as I've removed the documenting of the `log_vc_info` members anyway